includes/admin/views/html-pay-order.php

```diff
@@ -0,0 +1,77 @@
+                    <button
+                            class="modal-close modal-close-link dashicons dashicons-no-alt">
+                        <span class="screen-reader-text">Close modal panel</span>
+                    </button>
+                </header>
+                <article>
+                    <form id="wc-globalpayments-pay-order-form">
+						<?php wp_nonce_field( 'woocommerce-globalpayments-pay', 'woocommerce-globalpayments-pay-nonce' ); ?>
+                        <input type="hidden" name="woocommerce_globalpayments_pay" value="1"/>
+                        <input type="hidden" name="entry_mode"
+                               value="<?php echo \GlobalPayments\Api\Entities\Enums\ManualEntryMethod::MOTO; ?>"/>
+                        <input type="hidden" name="order_id" value="<?php echo $order->get_id(); ?>"/>
+                        <input type="hidden" name="order_key" value="<?php echo $order->get_order_key(); ?>"/>
+                        <div class="payment_methods" style="display: none">
+                            <input type="radio" class="input-radio" name="payment_method"
+                                   value="<?php echo $this->id; ?>" checked="checked">
+                        </div>
+						<?php $this->environment_indicator(); ?>
+                        <div class="wc_payment_method payment_method_<?php echo $this->id; ?> payment_box">
+                            <ul class="woocommerce-SavedPaymentMethods wc-saved-payment-methods"
+                                data-count="{{ data.payment_methods.length }}">
+                                <#if( data.payment_methods.length ){#>
+                                <#_.each( data.payment_methods, function( method ){#>
+                                <li class="woocommerce-SavedPaymentMethods-token">
+                                    <input id="wc-<?php echo $this->id; ?>-payment-token-{{ method.id }}"
+                                           type="radio"
+                                           name="wc-<?php echo $this->id; ?>-payment-token"
+                                           value="{{ method.id }}"
+                                           style="width:auto;"
+                                           class="woocommerce-SavedPaymentMethods-tokenInput"<#if( method.is_default
+                                    ){#> checked="checked"<#}#>/>
+                                    <label for="wc-<?php echo $this->id; ?>-payment-token-{{ method.id }}">{{
+                                        method.display_name }}</label>
+                                </li>
+                                <#})#>
+                                <#}#>
+								<?php
+								echo $this->get_new_payment_method_option_html();
+								$this->form();
+								?>
+                            </ul>
+                        </div>
+						<?php
```

Comment:
One way to simplify this is the following:

<?php if  ( ! empty( $order->get_transaction_id() ) ): ?>

And then have <?php endif; ?> on line 62 (61 after the refactoring).

This will reduce only one line of code, which is not too much, so we can also keep it the way it is now.